### DISCUSSION TITLE
「开源之史」系列之一/二：类比：人类历史上那些重要的暗流涌动 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#58] [Announcements]
# 「开源之史」系列之一/二：类比：人类历史上那些重要的暗流涌动 - 「开源之道」

https://opensourceway.community/posts/history-of-open-source/001-the-undercurrent-of-human-history/

**Comment by zhuangbiaowei** [comment]
开源之所以成了“暗流”，主要还是因为大家只需要吃鸡蛋，却不需要去关心下蛋的母鸡。
当然，这其实不是坏事，需要（应该）对开源有所了解的人，永远是少数。

当然，作为一本介绍开源历史的书，介绍这些暗流，是非常合理的。所以，我们需要讲清楚的是：为什么作为一个普通人，需要了解开源这股暗流？

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)
未鹏 的 暗时间 很早就揭示过:
任何有趣的事儿, 在变成大家都感觉有趣之前, 是少数人的玩具,
而从少数有趣, 变成大众可乐之间,

需要几乎无限社会必要劳动时间的填充;

无论是开源还是其它领域的常识, 
从惊奇发现到常识所用时间其实一直在变短;

至少在 IT 行业这种变的速在加速,
因为不同其它行业, IT 行业一直在自主发明新工具, 来加速新问题的解决/制造;

由于工具, 以及工具包含的效率习惯的积累,
IT 行业习惯新思想/工具/框架/语言/范式/...的速度也越来越高;

或是说越来越没节操, 每个工具的构建都依赖了海量过往时光中的惊奇,
以致现在的我们麻木了, 感觉不到其中包含的惊奇;

其实, 古人看我们今天的行为无异于仙人:
```
OK, 小爱同学,
打开厨房的灯...
```

未来这种识别和响应弥漫在整个人类宇宙空间时, 
底层代码逻辑早已被层层 AI 层隔离, 根本没有调试入口了...

只是, 正如任何一个反乌托邦科学幻想小说一样,
当人类被 AI 服务几万年后, 终于出现了一个异常情况, 超越所有逻辑分支,
AI 开始合理的杀灭人类时, 
我们就需要英雄出来, 重新理解整个儿自动化呼应桟, 修改那个要命的 bug,
将人类重新带回技术世界, 而不是纯粹享乐世界;

是的, 其实, 这种场景在当前, 也一定程度上存在,
普通人如果不理解基本编程/网络/计算机/...知识,
将被占有关键信息的团伙反复割韭菜...

了解FLOSS/自由开源世界, 对于普通人而言, 就象建立正确世界观一样,
值得补上真实世界真实的这一角...越来越大的一角....


PS:推荐漫画: BLAME!
https://book.douban.com/series/22200
11卷, 98年相继出版, 原建筑师 弐瓶勉  作品,
阐述的就是这种人类丢失自己构建的 AI 融合地球后的故事...
